speech/google/cloud/speech/streaming/request.py

```diff
@@ -0,0 +1,183 @@
+# Copyright 2016 Google Inc.
```

Comment:
This module defines no public members. It would've been better to have the module itself be non-public and then the members can be public within the non-public module.

(However, IMO you should condense the `streaming` package into one (or a couple) modules in the `speech` package, as I said above.)